src/applications/accredited-representative-portal/containers/POARequestsPage.jsx

```diff
@@ -1,17 +1,46 @@
 import React from 'react';
-
+import { useLoaderData, NavLink, Outlet } from 'react-router-dom';
 import DigitalSubmissionAlert from '../components/DigitalSubmissionAlert/DigitalSubmissionAlert';
-import POARequestsTableFetcher from '../components/POARequestsTableFetcher/POARequestsTableFetcher';
-import usePOARequests from '../hooks/usePOARequests';
 
 const POARequestsPage = () => {
+  const poaRequests = useLoaderData();
+
   return (
     <>
       <h1 data-testid="poa-requests-heading">Power of attorney requests</h1>
       <DigitalSubmissionAlert />
       <div className="poa-requests-page-table-container">
-        <h2 data-testid="poa-requests-table-heading">Requests</h2>
-        <POARequestsTableFetcher usePOARequests={usePOARequests} />
+        {poaRequests.length === 0 ? (
+          <p data-testid="poa-requests-table-fetcher-no-poa-requests">
+            No POA requests found
+          </p>
+        ) : (
+          <>
+            <div role="tablist" className="poa-request__tabs">
+              <NavLink
+                to="/poa-requests/pending"
+                className="poa-request__tab-link"
+                role="tab"
+                aria-selected={({ isActive }) => (isActive ? 'true' : 'false')}
```

Comment:
I don't see that this is one of the props that `NavLink` will accept a function for.

Reply:
you are right - it is not reflecting the correct state. I grabbed this from the react router docs and figured it would work for aria-selected.I _thought_ I also saw this on other files as well, but will double check the implementation

Reply:
re: the font comment - while working on accept/decline flow ticket, I also see that the fonts are not loading for me.